docs/reference-guides/block-api/block-transforms.md

```diff
@@ -221,7 +221,7 @@ transforms: {
 }
 ```
 
-<h4 id="schemas-and-content-models">Schemas and Content Models</h4>
+#### Schemas and Content Models
```

Comment:
Are the anchors auto-generated? Looks like this was added by @dmsnell in #36839.

Reply:
I think they were not auto-generated.

Reply:
I may have done this to create a fixed id for the header, as not all Markdown flavors support doing so.

Note that this change likely breaks the link above in line 197

Reply:
The link on line 197 still seems to work well. If we follow this link  https://github.com/WordPress/gutenberg/blob/docs/replace-and-html-usage-with-pure-markdown/docs/reference-guides/block-api/block-transforms.md#schemas-and-content-models we still go to the right section.

Reply:
very good. we don't have any way to prevent someone else breaking that when they change the text of the heading? if anything, that is why I manually added the HTML as a safeguard - because right now, the fact that the link works is accidental in Markdown at best. When someone undoubtedly comes in and changes the text of the heading, we want that link to remain.